content/en/developers/integrations/log_pipeline.md

```diff
@@ -16,111 +16,159 @@ further_reading:
+* To maintain optimal grok parsing performance, avoid wildcard matchers such as `%{data:}` and `%{regex(".*"):}`. Make your parsing statements as specific as possible.
+* Take the free course [Going Deeper with Logs Processing][20] for an overview on writing processors and leveraging standard attributes. 
 
-### Facet requirements
+### Create facets
 
-You can optionally create [facets][12] in the [Log Explorer][16]. Facets are specific attributes that can be used to filter and narrow down search results. While facets are not strictly necessary for filtering search results, they play a crucial role in helping users understand the available dimensions for refining their search.
+Facets are specific qualitative or quantitative attributes that can be used to filter and narrow down search results. While facets are not strictly necessary for filtering search results, they play a crucial role in helping users understand the available dimensions for refining their search. 
 
-Measures are a specific type of facet used for searches over a range. For example, adding a measure for latency duration allows users to search for all logs above a certain latency. 
-**Note**: Define the [unit][11] of a measure facet based on what the attribute represents.
+Facets for standard attributes are automatically added by Datadog when a pipeline is published. Review if the attribute should be remapped to a [Datadog Standard Attribute][6] instead. 
 
-To add a facet or measure:
+Not all attributes are meant to be used as a facet. The need for facets in integrations is focused on two things:
+* Facets provide a straightforward interface for filtering logs. They are leveraged in Log Management autocomplete features, allowing users to find and aggregate key information found in their logs.
+* Facets allow for attributes with low readability to be renamed with a label that is easier to understand. For example: `@deviceCPUper` → `Device CPU Utilization Percentage`.
+
+You can create [facets][12] in the [Log Explorer][16].
+
+**Add a facet or measure**:
+
+<div class="alert alert-info">Quantitative facets, called "Measures", allow users to filter logs over a range of numeric values using relational operators.  
+For example, a measure for a latency attribute allows users to search for all logs greater-than a certain duration. </div>
 
 1. Click on a log that contains the attribute you want to add a facet or measure for. 
 2. In the log panel, click the Cog icon next to the attribute.
 3. Select **Create facet/measure for @attribute**.
 4. For a measure, to define the unit, click **Advanced options**. Select the unit based on what the attribute represents.
+**Note**: Define the [unit][11] of a measure based on what the attribute represents.
 5. Click **Add**.
 
-To help navigate the facet list, facets are grouped together. For fields specific to the integration logs, create a **single group with the same name** as the `source` tag. 
```

Comment:
Can we keep this? All log integrations should have their own group for their custom facets

Reply:
I see that this is mentioned below. I think we might need to move this? If a partner creates a facet based on the instructions here then they won't know to add this to the integration group unless they keep reading. I'd either keep the group instructions here or move this after requirements? For example :

1. Facet overview
2. Requirements
3. Create facet instructions
4. Configure and edit

Reply:
I think visually, the requirements read-out can be a bit challenging initially - getting them to do the basic creation of the facet then have it fit the requirements might be easier.

I can add a quick "Facet group" instruction here and point the user down the docs for additional requirements on that.
So basically have this be step 5.
> To help navigate the facet list, facets can be grouped together. If a Group does not already exist for your Facets please read For fields specific to the integration logs, create a **single group with the same name** as the `source` tag.

I'll recommit and we can review if this is needed then.

Reply:
I've added this in a new commit.